### PR TITLE
add remove_data_bags, Fix issue #242

### PR DESCRIPTION
**Pull request by halida**
Fix issue #242

### CODE REVIEW

File: littlechef/runner.py [hunk 1]
```diff
@@ -444,6 +444,11 @@ def _readconfig():
     except (ConfigParser.NoOptionError, ConfigParser.NoSectionError):
         env.https_proxy = None
 
+    try:
+        env.remove_data_bags = config.get('userinfo', 'remove_data_bags')
+    except ConfigParser.NoOptionError:
+        env.remove_data_bags = None
```
Comment (tobami) [inline littlechef/runner.py]: In this case it would be better to default to `False`

**Comment by tobami** [comment]
Otherwise looks good!

**Comment by halida** [comment]
Change None to False.

**Comment by tobami** [comment]
Great, thanks!

**Pull request merged**